src/Domain/Messenger/MessengerGateway.php

```diff
@@ -788,16 +788,18 @@ public function getMessages(string $mode = 'print', string $date = '')
                     });
 
                 $table->addColumn('message', __('Message'))
-                    ->width('640px')
-                    ->addClass('align-top')
+                    ->width('650px')
+                    ->addClass('align-top overflow-x-scroll max-w-lg')
                     ->format(function ($message) {
+                        //$output = '<div style="max-width:600px;overflow-x: scroll;">';
```

Comment:
Ah, just noticed that this commented out code seems to have made its way into the commit. Can you clean these up? Thanks.
```suggestion
```